### PR TITLE
kola: add flannel smoke test

### PR DESCRIPTION
**Pull request by mischief**
wip. don't merge yet.

### CODE REVIEW

**Comment by mischief** [comment]
no longer wip. passes on aws. untested on gce. will not work in qemu due to lack of internet access.

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by pbx0** [comment]
lgtm

File: kola/flannelregistry.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      drop-ins:
+        - name: 50-network-config.conf
+          content: |
+            [Service]
+            ExecStartPre=/usr/bin/etcdctl set /coreos.com/network/config '{ "Network":"10.254.0.0/16", "Backend":{"Type": "vxlan"} }'
+      command: start
+    - name: docker.service
+      command: start
+`
+)
```
Comment (eyakubovich) [inline kola/flannelregistry.go]: The cloud-configs for `udpConf` and `vxlanConf` are almost identical. Maybe use `text/template` to reduce the duplication?

**Comment by eyakubovich** [comment]
LGTM other than code dup.

File: kola/flannelregistry.go (hunk 1 above)
Comment (mischief) [inline kola/flannelregistry.go]: yes.. this is desirable. but would require some integration with all the other uses of cloudconfigs in test specifications.

Comment (eyakubovich) [inline kola/flannelregistry.go]: Why? It's passed in `init()` -- can't you just render the template then?

**Comment by mischief** [comment]
@eyakubovich PTAL. use text/template now.

**Comment by eyakubovich** [comment]
LGTM

**Pull request merged**